src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -448,7 +447,7 @@ export function MultiSelect<T>({
             <IconButton
                 disabled={disabled}
                 name="operation-clear"
-                data-test-id="clear-all-items" // @deprecated should be created from the given testid plus `_clearance` suffix
+                data-test-id={dataTestId ?? dataTestid ? `${dataTestId ?? dataTestid}_clearance` : undefined}
```

Comment:
We need to postpone this change to the moment when we plan to release a new major version, for the upcoming minor version we need to keep the old test id.